### PR TITLE
Deliver events to a particular path of a Trigger subscriber

### LINKED ISSUE #2759: Document trigger.spec.subscriber type and usage with a particular path
In this page https://knative.dev/development/eventing/triggers/ we don't explain how
to send events to Trigger's subscriber to a particular path using `spec.subscriber.uri` 
or a combination of `spec.subscriber.ref` and `spec.subscriber.uri`.

## Additional Info

- https://pkg.go.dev/knative.dev/pkg/apis/duck/v1?tab=doc#Destination

### PR DESCRIPTION
**Pull request by pierDipi**
Fixes #2759

## Proposed Changes 

- Document trigger.spec.subscriber type and usage with a particular path

### CODE REVIEW

File: docs/eventing/triggers/README.md [hunk 1]
```diff
@@ -19,6 +21,26 @@ spec:
 EOF
 ```
 
+Simple example which will receive all the events from the given (`default`) broker and
+deliver them to Kubernetes service `my-service` to a custom path `/my-custom-path`:
```
Comment (mpetason) [inline docs/eventing/triggers/README.md:25]:
```suggestion
Simple example which will receive all the events from the given (`default`) broker and
deliver them to the custom path `/my-custom-path` for the Kubernetes service `my-service`:
```

Comment (pierDipi) [reply docs/eventing/triggers/README.md:25]: Thanks, fixed!

**Comment by vaikas** [comment]
/approve
Thanks for doing this!!!

**Pull request merged**